packages/core/src/rules/oas2/response-contains-property.ts

```diff
@@ -0,0 +1,23 @@
+import { Oas2Rule } from '../../visitors';
+
+export const ResponseContainsProperty: Oas2Rule = (options) => {
+  const mustExist = options.mustExist || [];
+  return {
+    Response: {
+      skip: (_response, key) => {
+        return !['200', '201', '202'].includes(key.toString());
```

Comment:
What do these numbers mean?

Reply:
Well, they may have response content but may be errors, and that's why they aren't applicable either.

204 - no content
4XX - bad requests
5XX - server errors

It's common that things like 4XX have different properties.

Reply:
Same as https://github.com/Redocly/openapi-cli/pull/537#discussion_r795133078

But here also different mime types are in play, so I'm not sure what to suggest yet.

@adamaltman do you have some ideas?

Reply:
Yes, I think the 2XX, 4XX configuration options is a good idea.